_posts/0000-01-02-MarcusVHarris.md/new file

```diff
@@ -0,0 +1 @@
+dog<- paste("dog")
```

Comment:
Cool so I guess I'd suggest the change be from (dog<-paste("dog")) to (cat<-paste("cat"))